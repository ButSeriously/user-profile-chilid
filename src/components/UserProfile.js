import React from 'react';
import UserHeader from './UserHeader';
import UserComments from './UserComments';

const UserProfile = () => (
  <div className="user_profile">
    <div className="user_profile__background" />
    <div className="user_profile__wrapper" />
    <UserHeader />
    <UserComments />
  </div>
);

export default UserProfile;
