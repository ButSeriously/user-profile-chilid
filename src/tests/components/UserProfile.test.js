import React from 'react';
import { shallow } from 'enzyme';
import UserProfile from '../../components/UserProfile';

test('should render UserProfile correctly', () => {
  const wrapper = shallow(<UserProfile />);
  expect(wrapper).toMatchSnapshot();
});
