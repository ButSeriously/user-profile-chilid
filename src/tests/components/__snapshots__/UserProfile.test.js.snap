// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render UserProfile correctly 1`] = `
<div
  className="user_profile"
>
  <div
    className="user_profile__background"
  />
  <div
    className="user_profile__wrapper"
  />
  <UserHeader />
  <UserComments />
</div>
`;
